mindmeld/converter/dialogflow.py

```diff
@@ -259,13 +260,20 @@ def _create_intent_file(
                 sentence += part
             all_text.append(sentence)
 
-        train, test = train_test_split(all_text, test_size=0.2)
+        for key in default_intent_to_training_file:
+            if key in mindmeld_intent_directory:
+                with open(os.path.join(package_dir, default_intent_to_training_file[key])) as fp:
+                    for line in fp:
+                        all_text.append(line.strip())
 
-        target_test.write("\n".join(test))
-        target_train.write("\n".join(train))
+        # Double the size of the training set if there are less than 10 training examples.
+        # This is needed since the k-fold cross validation parameter is set to 10 for
+        # intent classification.
```

Comment:
We could reasonably set `k` to 3 or 5 for this converter. `k=10` is good for when the training set N>50 or so, but using the minimum k is better than duplicating data. If the data is still lower than k, then yeah we need to duplicate.

Reply:
Understood, I will address this in a followup PR.